### ISSUE TITLE
Inaccessible elements

### ISSUE THREAD

**Issue by RobQuincey-DC** [Dorset-Council-UK/GIFramework-Maps#273]
Lighthouse reports 2 inaccessible elements on the main map page, due to lack of discernible text

![image](https://github.com/Dorset-Council-UK/GIFramework-Maps/assets/109682469/55c8010e-3919-4562-b297-9ad59bc41486)

We should look for additional ones that it did not pick up at the same time
